process-automation/latest/model/connectors/content.md

```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
 | --------- | ---- | ----------- |
 | response | JSON | *Optional.* The response from the call declaring the record. |
 
+## Security Marks
+
+The **ADD_SECURITY_MARKS** action is used to add security marks to specific content.  
+To add security marks in Process Automation you must have configured security marks in Governance Services. For information on security marks in Governance Services, see [Security Marks and Classification
+]({% link governance-services/latest/using/smc.md %})
+
+The input parameters from add security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| file | file | *Required.* ACS file to be updated. File variables are initialised by content actions or content events in triggers. E.g: 'Initialise a file variable by mapping it to the output of the generate document task. |
```

Comment:
When you refer to Content Services here you use ACS: "Required. ACS file to be updated." , should we stick to Alfresco Content Services as we are talking about another product, or I see some use of Content Services. But I think ACS could be confusing at first if you are not familiar with the abbreviation. 
This goes for all ACS usage below...

Reply:
Change "the generate document" to "the generated document"